examples/token-classification/run_tf_ner.py

```diff
@@ -203,8 +203,8 @@ def compute_metrics(p: EvalPrediction) -> Dict:
     trainer = TFTrainer(
         model=model,
         args=training_args,
-        train_dataset=train_dataset.get_dataset() if train_dataset else None,
-        eval_dataset=eval_dataset.get_dataset() if eval_dataset else None,
+        train_sized_dataset=(train_dataset.get_dataset(), len(train_dataset)) if train_dataset else (None, None),
```

Comment:
Why should the user provide the size of dataset by himself ?

Wasn't it better when it was done automatically ? Also, it means user needs to take care of batch size in order to compute the right dataset size, am I right ?